pkg/trace/config/apply.go

```diff
@@ -28,6 +28,10 @@ import (
 // apiEndpointPrefix is the URL prefix prepended to the default site value from YamlAgentConfig.
 const apiEndpointPrefix = "https://trace.agent."
 
+// telemetryEndpointPrefix is the URL prefix for Instrumentation telemetry endpoint
+const telemetryEndpointPrefix = "https://instrumentation-telemetry-intake."
+const defaultSite = "datadoghq.com"
```

Comment:
Delete this please and use `(pkg/config).DefaultSite`

Reply:
👍 Was looking for previously defined default site, and couldn't find it. 🙈 - On it

Reply:
Done